.github/workflows/ci.yml

```diff
@@ -446,7 +446,7 @@ jobs:
         if: always()
         uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4
         with:
-          name: unit-tests-frontend-junit
+          name: optimize-unit-tests-frontend-junit
```

Comment:
is this an unrelated fix?

Reply:
Sorry - not a fix and not related, just noticed the name for this was a bit unspecific